core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.spec.ts

```diff
@@ -183,6 +188,46 @@ describe('Translation language selector', () => {
     expect(component.activeLanguageCode).toBe('en');
   });
 
+  it('should display the preferred language when the preferred' +
+    ' language is defined', async(() => {
+    component.activeLanguageCode = null;
+    preferredLanguageCode = 'en';
+    // This throws "TS2532". We need to suppress this error because
+    // "constants.SUPPORTED_AUDIO_LANGUAGES" is always defined. Also, it must
+    // contain description for language code 'en' and if it doesn't
+    // then this test should fail.
+    // @ts-ignore
```

Comment:
Is this ignore still needed?

Reply:
Actually, I am getting a "TS2523" error if I try to find language description from AppConstants like this: 
```
const languageDescription = AppConstants.SUPPORTED_AUDIO_LANGUAGES.find(
      e => e.id === 'en').description;
```
But now I have removed this ts-ignore comment by using optional chaining like this:
```
const languageDescription = AppConstants.SUPPORTED_AUDIO_LANGUAGES.find(
      e => e.id === 'en')?.description ?? '';
```